tuareg.el

```diff
@@ -3645,35 +3682,43 @@ It is assumed that the range START-END delimit valid OCaml phrases."
 (defun tuareg-eval-region (start end)
   "Eval the current region in the OCaml REPL."
   (interactive "r")
-  (setq tuareg-interactive-last-phrase-pos-in-source start)
-  (save-excursion
-    (goto-char start)
-    (tuareg-backward-beginning-of-defun)
-    (setq start (point))
-    (setq end (cdr (tuareg-region-of-defun end)))
-    (if end
-        (tuareg-interactive--send-region start end)
-      (message "The expression after the point is not well braced."))))
+  (tuareg-interactive--send-region start end))
```

Comment:
Souldn't `tuareg-interactive--send-region` be removed (it is an internal function) and its code be put in `tuareg-eval-region`?

Reply:
That's right, we can do that now as a final refactoring step. (It's a matter of taste whether you want to use an interactive command as subroutine for another function but that seems to be an old Emacs tradition, layering be damned.)